src/index.html

```diff
@@ -9,10 +9,64 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__wrapper">
+        <header class="product-card__header">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="product-card__img"
+          />
+        </header>
+        <section class="product-card__main product-info">
```

Comment:
```suggestion
        </header>
        
        <section class="product-card__main product-info">
```

Reply:
<img width="1116" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/86789161/110e5ecf-853f-4eac-bc70-092dadd7c51c">

Please, do it everywhere below

Reply:
ok